File: 2026/LABs/LAB01/maiusculo.c
#include <stdio.h>
#include <string.h>

int main(void){
    char palavra[100];
    int bts = 0; // contador

    scanf("%s",palavra);

    while(palavra != "FIM"){
        for(int i = 0; i < strlen(palavra); i++ ){
            if(palavra(i) >= 'A' && palavra(i) <= 'Z'){
                bts++;
            }
        }
        printf("%d",bts);
        bts = 0;
        scanf("%s",palavra);
    }

    return 0;
}

File: 2026/LAB01/Maiusculo.java
import java.util.Scanner;

public class Maiusculo {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int bts; //contador
        bts = 0;
        String palavra;
        palavra = sc.nextLine();

        while(palavra != "FIM"){
            for(int i = 0; i < palavra.length(); i++){
                if(palavra.charAt(i) >= 'A' && palavra.charAt(i) <= 'Z'){
                        bts++;
                }
                
            }

            System.out.println(bts);
            bts = 0;
            palavra = sc.nextLine();
        }
        sc.close();
        }
        
}
